tests/chainer_tests/test_cuda.py

```diff
@@ -32,16 +32,14 @@ def test_ne(self):
 
 class TestCuda(unittest.TestCase):
 
-    def test_get_dummy_device(self):
-        self.assertIs(cuda.get_device(), cuda.DummyDevice)
-
-    def test_get_device_for_numpy_int(self):
-        self.assertIs(cuda.get_device(numpy.int64(0)), cuda.DummyDevice)
+    def test_get_device_from_id_for_numpy_int(self):
+        self.assertIs(cuda.get_device_from_id(numpy.int64(0)),
+                      cuda.DummyDevice)
```

Comment:
Should it be `cuda.get_device_from_array`?
This is out of specification of  `cuda.get_device_from_id`. It should throw an error.